### DISCUSSION TITLE
Including TODOs from future daily note pages

### DISCUSSION THREAD

**Discussion by jenningsb2** [ahonn/logseq-plugin-todo#51] [Ideas]
Currently, when TODOs are on future daily note pages, they don't appear in the widget. It would be great if they appeared in the 'Anytime' or even a new section called 'Future.'

**Comment by ahonn** [comment]
Makes sense, I will find time recently to support this feature.
And I think add a new section called 'Future' is better.

**Comment by ahonn** [comment]
@jenningsb2 Hi, The latest version already shows these TODOs in the scheduled section
